engine/src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -6,8 +6,6 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.collection.immutable.Seq
 
 package object backend {
-  class WorkflowContext(val root: String)
```

Comment:
It's more avoiding the slow creep we previously saw. Feel free to disregard entirely